### ISSUE TITLE
Image field : Update doesn't show as expected

### ISSUE THREAD

**Issue by NasserBvB** [keystonejs/keystone#6554]
## Description of the bug
This bug just appeared in the production mode.
In the production, when I add a profile_pic field to the user schema. When updating the profile_pic, it performs the action successfully, but the image seems not be found in the public/images/xxxxx.png. However, the image exists in the images' directory. When I restart the server, the image is rendered successfully.
## Steps to reproduce the behavior

- Add config for images to the keystone config:
```typescript
export default withAuth(
  config({
    ..........
    images: {
      upload: "local",
      local: {
        storagePath: "public/images",
        baseUrl: "/images",
      },
    },
    .........
);
```
- Create a schema with an image field: 
```typescript
export const User = list({
....
     profile_pic: image({}),
....
})
```

## Expected behavior

I expected the image to be modified and displayed as I submit changes.

## Screenshots

Uploading the image: 
![uploading the image](https://user-images.githubusercontent.com/48838783/133266502-f55f589d-656a-4b03-9e24-13c76f3a04d0.png)

Submitting changes:
After submitting the changes, the image doesn't show properly .
![After submiting changes](https://user-images.githubusercontent.com/48838783/133266683-e0f50eb3-a2cd-406a-81c1-799780df7596.png)

When I restart the server, the image is rendered properly.
## System

- OSes: macOS, Linux Ubuntu 20.4;
- Browsers: chrome, safari;

## Additional context
The error appears only in the production mode, not in the development mode.

**Comment by doncazzo** [comment]
i saw the same, the poblem is that next only upload that images whwn you start the server, you have to reset the server or change the save method to save the image in the cloud or s3

**Comment by gautamsi** [comment]
this is wired bug, when you restart your server it will start showing files uploaded before restart

**Comment by bladey** [comment]
Thanks @NasserBvB, I can reproduce this, will investigate.

**Comment by NasserBvB** [comment]
Thank you @bladey.

**Comment by painttist** [comment]
I am experiencing the same problem. This is not just an admin-ui issue, directly accessing the newly uploaded image from the public link will also show 404 not found. Is there anyway update on how to fix this?

![chrome_2021-11-12_15-21-28](https://user-images.githubusercontent.com/50148803/141426696-c5920fef-46ab-4d53-bd7f-46fa42548ef8.png)
![chrome_2021-11-12_15-21-19](https://user-images.githubusercontent.com/50148803/141426690-71675a8c-d9e4-4037-bfb5-1ee201b00b27.png)

**Comment by painttist** [comment]
I think adding `extendExpressApp` as mentioned in #6686 by @bjoluc is a workaround.

But be careful to use `app.use` instead of `app.get`

in your  `config`
```javascript
export default withAuth(
  config({
    ...
    server: {
      extendExpressApp: (app, createContext) => {
        app.use(express.static("public"));
        // Alternatively you can restrict the access more with
        // app.use('/images', express.static("public/images"));
      },
    }
  })
);
```

**Comment by mitchellhamilton** [comment]
This was fixed by #6893 which is available in `@keystone-next/keystone@27.0.2` and above.
